src/thread.c

```diff
@@ -177,12 +180,22 @@ static rt_err_t _thread_init(struct rt_thread *thread,
     /* init thread stack */
     rt_memset(thread->stack_addr, '#', thread->stack_size);
 #ifdef ARCH_CPU_STACK_GROWS_UPWARD
+#ifdef RT_USING_TLS
+    thread->tls = thread->stack_addr;
+    _init_tls(thread->tls);
```

Comment:
From my understanding, it's a porting layer for tls (in libc?). It's better to use a unify name for here.

Reply:
Agreed -- I will adjust this to use an rt-prefixed name instead of the internal picolibc name.

Reply:
I've pushed an updated version which creates an rt abstraction and uses the picolibc implementation for picolibc.

Reply:
And another update which installs the GNU Arm Embedded Toolkit version of picolibc for CI tests.